hibike/hibike_message.py

```diff
@@ -405,19 +407,21 @@ def parse_bytes(msg_bytes):
     return HibikeMessage(message_id, payload)
 
 
-def blocking_read_generator(serial_conn, stop_event=None):
+def blocking_read_generator(serial_conn, stop_event=threading.Event()):
```

Comment:
Interesting quirk with python arg defaults: 
One object is created when the module is imported, and that is used for every single function call. 

So when one threading.Event() for a blocking_read_generator called with default arguments triggers, every single blocking_read_generator called with default arguments will trigger. 

I recommend keeping it None, then have an `if stop_event == None` etc.

Reply:
This shouldn't be an issue, because the event is never triggered within `blocking_read_generator`,